raiden/api/mock.py

```diff
@@ -33,7 +33,7 @@ class MockAPI(object):
     """
     channel_by_address = dict() # 1:1
     all_channel = [] # N
-    channels_by_asset = defaultdict(list) # 1:N
+    channels_by_token = defaultdict(list) # 1:N
     channels_by_partner = defaultdict(list) # 1:N
```

Comment:
This whole file will probably go away. Kept it for manual testing, but now we got actual tests so not really needed.